site2/docs/getting-started-standalone.md

```diff
@@ -1,278 +1,150 @@
-
-:::
-
-## Use Pulsar standalone
-
-Pulsar provides a CLI tool called [`pulsar-client`](reference-cli-tools.md#pulsar-client). The pulsar-client tool enables you to consume and produce messages to a Pulsar topic in a running cluster. 
-
-### Consume a message
-
-The following command consumes a message with the subscription name `first-subscription` to the `my-topic` topic:
+Run this command to launch the consumer:
 
 ```bash
-bin/pulsar-client consume my-topic -s "first-subscription"
+bin/pulsar-client consume quickstart -s 'first-subscription' -p Earliest -n 0
 ```
 
-If the message has been successfully consumed, you will see a confirmation like the following in the `pulsar-client` logs:
+Earliest means consuming from the earliest **unconsumed** message. `-n` configures the number of messages to consume, 0 means to consume forever.
 
-```
-22:17:16.781 [main] INFO  org.apache.pulsar.client.cli.PulsarClientTool - 1 messages successfully consumed
-```
+As before, this is useful for trialling things on the command line, but in practice you'll use the Consumer API in your application code, or Pulsar IO connectors for reading data from Pulsar to push to other systems.
 
-:::tip
+You'll see the messages that you produce in the previous step:
 
-As you have noticed that we do not explicitly create the `my-topic` topic, from which we consume the message. When you consume a message from a topic that does not yet exist, Pulsar creates that topic for you automatically. Producing a message to a topic that does not exist will automatically create that topic for you as well.
+```text
+----- got message -----
+key:[null], properties:[], content:Hello Pulsar!
+```
 
-:::
+## Write some more messages
 
-### Produce a message
+Leave the consume command from the previous step running. If you've already closed it, just re-run it.
 
-The following command produces a message saying `hello-pulsar` to the `my-topic` topic:
+Now open a new terminal window and produce more messages, the default message separator is `,`:
 
 ```bash
-bin/pulsar-client produce my-topic --messages "hello-pulsar"
-```
-
-If the message has been successfully published to the topic, you will see a confirmation like the following in the `pulsar-client` logs:
-
-```
-22:21:08.693 [main] INFO  org.apache.pulsar.client.cli.PulsarClientTool - 1 messages successfully produced
+bin/pulsar-client produce quickstart --messages "$(seq -s, -f 'Message NO.%g' -t '\n' 1 10)"
 ```
 
-## Stop Pulsar standalone
+Note how they are displayed almost instantaneously in the consumer terminal.
 
-Press `Ctrl+C` to stop a local standalone Pulsar.
+## Stop the Pulsar cluster
 
-:::tip
+Once you've finished you can shut down the Pulsar cluster. Press **Ctrl-C** in the terminal window in which you started the cluster.
```

Comment:
Control + C?

Reply:
Ctrl-C and Ctrl+C read the same from a developer's perspective. No need to change.